### PR TITLE
Run get_json with an interval periodically in a thread

### LINKED ISSUE #76: Run get_json periodically
Add a duration in settings.py, and run get_json at that duration. This should happen async, so that it does not affect the server.

### PR DESCRIPTION
**Pull request by an00bhav**
This is pretty complicated, and we _MUST_ move to celery as
Tyler mentioned. Some things that are to be noted is that
you will see the logger messages printed two times, as now
there are two instances of logger, ie the thread and the main
process, I don't think the code is running two times.

To kill the server and restart, Ctrl-C will no longer work, as there
are threads now. Do the following:

1) Find the process id of the task by running
`ps -ef | grep run_server`
2) Kill all the process using `kill -9 PID` where PID is the number
you found in 1.

Also auto_restart of flask will not work the way it should. Sorry
about it. Manually restart with the process mentioned above.

Fixes #76

### CODE REVIEW

**Comment by Mester** [comment]
I think before this is merged, add instructions on how to do that stuff somewhere that is not in the commit message/comment here. Possibly in the readme (you could even then solve #72 , two birds with one stone). Also merge in master beforehand so the conflicts are solved

File: music_app/__init__.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import logging
+import threading
+from time import sleep
 from flask import Flask
-from music_app.settings import LOGLEVEL
+from music_app.settings import LOGLEVEL, UPDATE_INTERVAL
+from music_app.get_json import main
 
 logger = logging.getLogger('music_app')
 logger.setLevel(getattr(logging, LOGLEVEL))
```
Comment (tylerphillips55) [inline music_app/__init__.py:9]: did you mean getattr(logger,...)?

Comment (an00bhav) [inline music_app/__init__.py:9]: No. It's logging. I am trying to call logging.DEBUG

Comment (tylerphillips55) [inline music_app/__init__.py:9]: then shouldn't it be (logging, DEBUG, LOGLEVEL)?

Comment (an00bhav) [inline music_app/__init__.py:9]: No. As I said I am calling logging.DEBUG where DEBUG is stored in LOG LEVEL which is coming from settings.py. So to call logging.DEBUG I need to call logging.LOGLEVEL, which I do using getattr

**Comment by an00bhav** [comment]
If we have problems with the threading, I'll give it a try to get celery working. Will take some time, as I'll be doing it after my office in the evening.

**Comment by an00bhav** [comment]
Have a better option now.

**Pull request closed without merging**